login creado

dsalazar31 opened the pull request.

dsalazar31 committed:
login creado

diegovaldezl commented on ChivoFlixDesktop/Data/MetodosCRUD.cs:
```diff
@@ -0,0 +1,62 @@
+            {
+                cmd = new SqlCommand();
+                cmd.Connection = cnn;
+                cmd.CommandType = CommandType.Text;
```
Se tiene que validar que solo personas con rol "Administrador" tengan acceso al sistema.

dsalazar31 committed:
validación del rol del usuario

dsalazar31 committed:
actualización daniel

Validación si el usuario es administrador, además se crean los roles al momento de crear la BD y sus tablas, sería bueno que por defecto se cree un usuario con rol de administrador para que se puede ingresar al sistema directamente
